crates/holochain_cascade/src/test_utils.rs

```diff
@@ -87,6 +88,14 @@ impl HolochainP2pDnaT for PassThroughNetwork {
         Ok(out)
     }
 
+    async fn get_by_op_type(
+        &self,
+        _action_hash: ActionHash,
+        _op_type: ChainOpType,
+    ) -> HolochainP2pResult<WireMaybeOpByType> {
+        todo!()
```

Comment:
If this is going to be committed, I would recommend an `unimplemented!` call with a comment instead of a todo.


```suggestion
        unimplemented!("Not supported in tests")
```